tests/nodeos_contrl_c_test.py

```diff
@@ -98,16 +98,16 @@
     testSuccessful=False
 
     Print("Configure and launch txn generators")
-    nonProducerP2pPort = cluster.getNodeP2pPort(nonProdNode.nodeId)
     targetTpsPerGenerator = 500
     testTrxGenDurationSec=60
     trxGeneratorCnt=1
-    nonProdNode.launchTrxGenerators(tpsPerGenerator=targetTpsPerGenerator, numGenerators=trxGeneratorCnt, durationSec=testTrxGenDurationSec,
-                                    contractOwnerAcctName=cluster.eosioAccount.name, acctNamesList=[accounts[0].name,accounts[1].name],
-                                    acctPrivKeysList=[account1PrivKey,account2PrivKey], p2pListenPort=nonProducerP2pPort, waitToComplete=False)
+    cluster.launchTrxGenerators(contractOwnerAcctName=cluster.eosioAccount.name, acctNamesList=[accounts[0].name,accounts[1].name],
+                                acctPrivKeysList=[account1PrivKey,account2PrivKey], nodeId=nonProdNode.nodeId, tpsPerGenerator=targetTpsPerGenerator,
+                                numGenerators=trxGeneratorCnt, durationSec=testTrxGenDurationSec, waitToComplete=False)
 
     Print("Give txn generator time to spin up and begin producing trxs")
-    time.sleep(10)
+    cluster.waitForTrxGeneratorsSpinup(nodeId=nonProdNode.nodeId, numGenerators=trxGeneratorCnt)
+    time.sleep(5)
```

Comment:
This sleep is no longer needed, right?

Reply:
Addressed: [de3a1c8](https://github.com/AntelopeIO/leap/pull/639/commits/de3a1c8392564d6d0ab41412374948c4c8f5359c)

Reply:
It was just waiting before to make sure trx were flowing. So I think you can remove the sleep with your better wait for first trx.

Reply:
Actually can't do the wait here due to all production nodes being shut down.